pkgs/top-level/php-packages.nix

```diff
@@ -92,13 +94,27 @@ let self = with self; {
     buildInputs = [ pkgs.m4 ];
   };
 
-  pthreads = assert pkgs.config.php.zts or false; buildPecl {
-    #pthreads requires a build of PHP with ZTS (Zend Thread Safety) enabled
-    #--enable-maintainer-zts or --enable-zts on Windows
+  #pthreads requires a build of PHP with ZTS (Zend Thread Safety) enabled
+  #--enable-maintainer-zts or --enable-zts on Windows
+  pthreads = if php7 then pthreads31 else pthreads20;
+
+  # 2.x is not supported and does not work on PHP 7
+  pthreads20 = assert pkgs.config.php.zts && (!php7); buildPecl {
```

Comment:
You want `(pkgs.config.php.zts or false)`, not `pkgs.config.php.zts` here (and below). Otherwise if `zts` isn't defined this will throw an eval error (note the `or false` in the original pthreads)